tools/NuGet/template-nuget/DynamoVisualProgramming.WpfUILibrary.nuspec

```diff
@@ -10,27 +10,32 @@
         <icon>content\images\logo.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Build node libraries for Dynamo with custom UI in WPF. It contains the following files:
-        1) DynamoCoreWpf.dll
-        2) CoreNodeModels.dll
-        3) CoreNodeModelsWpf.dll</description>
+        1) DynamoCoreWpf.dll </description>
         <summary>This package contains all that is required to get up and running building nodes for the Dynamo Visual Programming language with custom UI in WPF.</summary>
-        <copyright>Copyright Autodesk 2020</copyright>
+        <copyright>Copyright Autodesk 2023</copyright>
         <frameworkAssemblies>
-            <frameworkAssembly assemblyName="System.Xaml"/>
-            <frameworkAssembly assemblyName="WindowsBase"/>
-            <frameworkAssembly assemblyName="PresentationCore"/>
-            <frameworkAssembly assemblyName="PresentationFramework"/>
+            <frameworkAssembly assemblyName="System.Xaml" targetFramework="net48"/>
+            <frameworkAssembly assemblyName="WindowsBase" targetFramework="net48"/>
+            <frameworkAssembly assemblyName="PresentationCore" targetFramework="net48"/>
+            <frameworkAssembly assemblyName="PresentationFramework" targetFramework="net48"/>
         </frameworkAssemblies>
         <dependencies>
             <group targetFramework=".NETFramework4.8">
                 <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodes" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodesWPF" version="$Version$"/>
+            </group>
+            <group targetFramework="net6.0">
+                <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodes" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodesWPF" version="$Version$"/>
             </group>
         </dependencies>
     </metadata>
+    <!--for now we assume that that the working directory root is dynamo/bin/AnyCPU/Release-->
     <files>
         <file src="DynamoCoreWpf.dll" target="lib\net48" />
-        <file src="nodes\CoreNodeModels.dll" target="lib\net48" />
-        <file src="nodes\CoreNodeModelsWpf.dll" target="lib\net48" />
+        <file src="..\..\NET60_Windows\Release\DynamoCoreWpf.dll" target="lib\net6.0" />
```

Comment:
Is there an analagous nuget package like this one for headless applications that references DynamoCore and DynamoCoreNodes packages?

Reply:
No, I don't think so, I will add it to the future use cases to support. I will be filing some followup tasks for sure - because I think at the moment even the current DVP.Core package won't work 100% for a true headless integration.